### PR TITLE
Update onnx nlp and obj examples

### PR DESCRIPTION
**Pull request by yuwenzho**
## Type of Change

update examples
API not changed

## Description

Migrate ONNX nlp and obj examples to new API
JIRA ticket: [ILITV-2722](https://jira.devtools.intel.com/browse/ILITV-2722)

## Expected Behavior & Potential Risk

ONNX nlp and obj examples with new API run sucessufully.

## How has this PR been tested?

example extension test

## Dependency Change?

No.

### CODE REVIEW

**Comment by yuwenzho** [comment]
nlp extension test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4498/

**Comment by yuwenzho** [comment]
> obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4533/ nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4534/

Reasons for acc&perf regression: 
1. default graph optimization is ’basic‘ instead of ‘extended‘ now. 
2. graph optimization for nlp model is automatically set to 'extended'

Rerun failed examples: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4539/

**Comment by yuwenzho** [comment]
obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4578/

**Comment by yuwenzho** [comment]
obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4595/
nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4597/

**Comment by yuwenzho** [comment]
obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4598/
nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4599/

**Comment by yuwenzho** [comment]
obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4623/
nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4624/

**Comment by yuwenzho** [comment]
> obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4623/ nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4624/

new nlp example: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4630/

**Comment by yuwenzho** [comment]
> > obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4623/ nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4624/
> 
> new nlp example: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4630/

rerun failed examples: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4632/

**Comment by yuwenzho** [comment]
> > > obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4623/ nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4624/
> > 
> > 
> > new nlp example: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4630/
> 
> rerun failed examples: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4632/

fix doamin detection for gpt2: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4639/

**Comment by yuwenzho** [comment]
obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4643/artifact/report.html
nlp: obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4644/artifact/report.html

**Comment by yuwenzho** [comment]
> obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4643/artifact/report.html nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4644/artifact/report.html

rerun failed hf question answering task: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4649/

**Review by mengniwang95 — APPROVED** [review]

**Comment by yuwenzho** [comment]
> > obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4643/artifact/report.html nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4644/artifact/report.html
> 
> rerun failed hf question answering task: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4649/

add verision check for optimize_model function
re-test nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4684/

**Comment by yuwenzho** [comment]
> > > obj: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4643/artifact/report.html nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4644/artifact/report.html
> > 
> > 
> > rerun failed hf question answering task: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4649/
> 
> add verision check for optimize_model function re-test nlp: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4684/

rerun failed nlp example: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4688/

File: examples/README.md [hunk 1]
```diff
@@ -1005,55 +1005,55 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>*BERT base MRPC</td>
```
Comment (chensuyue) [inline examples/README.md:1005]: remove the `*` for new API models.

**Pull request merged**